### PR TITLE
Capitalization Update in Header

### PR DESCRIPTION
**Pull request by mhmohona**
I have noticed in most headline `Title Case` is used. However, in some cases `Title Case` was not used. This PR implements a `Title Case` capitalization update across various header in the documentation to enhance readability and consistency in documentation headers.

While making the changes, I didnt change the case for technical terms.

### CODE REVIEW

**Comment by mattlord** [comment]
LGTM, thanks @mhmohona ! Can you resolve the conflict? I can then approve and merge.

**Comment by mhmohona** [comment]
@mattlord I have done so. Could you please have a look?

File: content/en/docs/21.0/reference/features/vitess-sequences.md [hunk 1]
```diff
@@ -145,6 +145,7 @@ Now any table that will be using the new sequence can reference it in its VSchem
 ```
```
Comment (mattlord) [inline content/en/docs/21.0/reference/features/vitess-sequences.md:145]: We seem to be deleting content in this file.

File: content/en/docs/21.0/reference/programs/vtctldclient/_index.md
Comment (mattlord) [inline content/en/docs/21.0/reference/programs/vtctldclient/_index.md:1]: The vtctldclient reference docs are built from source so all of these changes will be lost. The changes would need to be made in the source and then the reference docs rebuilt (see the Makefile).

The tools used are the internal docgen tool: https://github.com/vitessio/vitess/blob/main/go/cmd/internal/docgen/docgen.go Which leverages cobra's doc tool: https://pkg.go.dev/github.com/spf13/cobra@v1.8.1/doc

Comment (mattlord) [reply content/en/docs/21.0/reference/programs/vtctldclient/_index.md:1]: This makes me think that perhaps we should go in the other direction and standardize around this casing (only capitalizing the first word and of course any proper nouns).

**Review by mattlord — CHANGES_REQUESTED** [review]

File: content/en/docs/21.0/reference/features/vitess-sequences.md (hunk 1 above)
Comment (mhmohona) [reply content/en/docs/21.0/reference/features/vitess-sequences.md:145]: Hey @mattlord, thank you so much for going through all the changes I made. 
Here the code block is intact. As you can see in preview - https://deploy-preview-1824--vitess.netlify.app/docs/21.0/reference/features/vitess-sequences/#creating-a-sequence

and in actual file - https://github.com/mhmohona/vitessio-website/blob/headline-fix/content/en/docs/21.0/reference/features/vitess-sequences.md?plain=1#L125

File: content/en/docs/21.0/reference/programs/vtctldclient/_index.md
Comment (mhmohona) [reply content/en/docs/21.0/reference/programs/vtctldclient/_index.md:1]: Yea, after changing some files, only then I noticed that these files are auto generated. 😅 Thanks for giving me details. I will check it.

**Comment by deepthi** [comment]
I disagree with most of the changes to hand-written docs. There is no need to capitalize everything in those headings. As @mattlord pointed out the vtctld command reference docs are auto-generated, and any changes to those need to happen in the tool. I think we should close this PR, and decide whether it is worth changing docgen. If we want to change it, that would be a PR on vitessio/vitess.

**Comment by deepthi** [comment]
@mhmohona We appreciate that you are spending time on this. In future, it will be better to propose changes in an issue and get some feedback before you start making PRs. That will ensure that both your time and maintainer time is used wisely, and not wasted on PRs that ultimately don't get merged.

**Pull request closed without merging**